QR-dtg.py

```diff
@@ -57,7 +70,7 @@ def main(webhook_url) -> None:
     if not (div := re.search(r"qrCode-......", str(source))):
         print(f"{Fore.LIGHTRED_EX}Error: \
 the regular expression 'qrCode-......' is not found.")
-        os._exit(1)
+        sys.exit()
```

Comment:
`os._exit():`
Exit the process with status n, without calling cleanup handlers, flushing stdio buffers, etc.

**Note** The standard way to exit is sys.exit(n). _exit() should normally only be used in the child process after a fork().